<!DOCTYPE html>
<html lang="en">
  <head>
    <!-- Basic Page Needs
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
    <meta charset="utf-8" />
    <title>James Pease</title>
    <meta name="description" content="James Pease website" />
    <meta name="author" content="James Pease" />
    <!-- Mobile Specific Metas
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <!-- FONT
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
    <!-- CSS
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
    <link rel="stylesheet" href="css/normalize.css" />
    <link rel="stylesheet" href="css/skeleton.css" />
    <link rel="stylesheet" href="css/main.css" />
	<link rel="stylesheet" href="font/fonts.css" />
    <!-- Favicon
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
    <link rel="icon" type="image/png" href="images/favicon.png" />
  </head>
  <body>
      <label for="lightswitch" id="lslabel">l</label>
      <input type="checkbox" id="lightswitch" class="toggle" />

    <!-- Primary Page Layout
        ================================================== -->
    <div class="container" id="main">
      <div class="row">
        <div id="nameheader" class="eleven columns">James Pease</div>
     </div>
      <div id="header" class="row">
        <div class="six columns">
          <div id="subheader">Postdoctoral Fellow
          <br />
          <a href="http://lsa.umich.edu/eeb/" target="_blank">Ecology &amp; Evolutionary Biology</a>
          &middot;
          <a href="http://www.umich.edu" target="_blank">University of Michigan</a><br />
			<a href="http://www.blackrim.org/" target="_blank">SA Smith Lab</a> &middot; 
          <a href="http://sites.lsa.umich.edu/cwdick-lab/" target="_blank">CW Dick Lab</a>
          &middot;
		  <a href="http://eas2.unl.edu/~sfritz/amazon/" target="_blank">Amazon Andes Geo-Genomics</a>
		  </div><br />
        </div>
        <div class="mainlink four columns">
          <p>
          <script type="text/javascript"><!--
var juyvepk = ['@','.','p','e','a','a','i','i','=','c','e','"',':','l','f','m','c','j','<','e','u','s','b','t','h','.','d','s','c','r','e','h','m','a','e','u','a','e','o','s','i','<',' ','a','h','s','=','u','>','d','"','e','@','l','u','"','m','p','a','a','b','l','>','/','"','j',' ','i','e','m'];var abovurq = [23,62,51,30,44,53,59,26,7,35,42,33,15,36,6,25,27,16,66,19,24,54,50,13,61,29,64,38,60,4,63,28,58,37,22,65,1,5,14,39,45,0,2,10,3,21,40,57,48,31,47,52,56,46,32,8,43,18,68,20,17,12,69,67,41,49,34,11,55,9];var uhqnltt= new Array();for(var i=0;i<abovurq.length;i++){uhqnltt[abovurq[i]] = juyvepk[i]; }for(var i=0;i<uhqnltt.length;i++){document.write(uhqnltt[i]);}
// --></script>
<noscript>Please enable Javascript to see the email address</noscript><br />
          <a href="http://www.twitter.com/jamesbpease">@jamesbpease</a>
          </div>
		  <div class="mainlink two columns">
          <a href="http://scholar.google.com/citations?user=VzP548QAAAAJ">G.Scholar</a> <br />
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/?%20term=Pease+JB%5BAuthor%5D">PubMed</a> <br />
          <a href="http://www.github.com/jbpease/">GitHub</a>
          <br />

          <br /></p>
        </div>
      </div>
      <div class="row">
          <div class="nine columns">
            <h3>News</h3>
            <ul class="news">
                <li> 2016-Feb-01: I will be teaching at the EMBO Computation Molecular Evolution Course in Crete this May.  Check <a href="http://events.embo.org/16-computational-evolution/">http://events.embo.org/16-computational-evolution/</a> for details.</li>
            </ul>
        
            <h3>Research</h3>
            <p>My research focuses on using multi-genome datasets to investigate the evolutionary histories of clades,
               especially processes of introgression, selection, molecular adaptation, and environmental effects that 
               occur alongside the processes of speciation and diversification. 
            </p>
          
          <h3>Publications</h3>
         <div class="pubtitle">
            <a href="#">
               Phylogenomics reveals three sources of adaptive variation during a rapid radiation. 
               <span>Speciation events often occur in rapid bursts of diversification, but the ecological and genetic factors that promote these radiations are still much debated. Using whole transcriptomes from all 13 species in the ecologically and reproductively diverse wild tomato clade (<em>Solanum</em> sect. <em>Lycopersicon</em>), we infer the species phylogeny and patterns of genetic diversity in this group. Despite widespread phylogenetic discordance due to the sorting of ancestral variation, we date the origin of this radiation to approximately 2.5 million years ago and find evidence for at least three sources of adaptive genetic variation that fuel diversification. First, we detect introgression both historically between early-branching lineages and recently between individual populations, at specific loci whose functions indicate likely adaptive benefits. Second, we find evidence of lineage-specific <em>de novo</em> evolution for many genes, including loci involved in the production of red fruit color. Finally, using a &ldquo;PhyloGWAS&rdquo; approach, we detect environment-specific sorting of ancestral variation among populations that come from different species but share common environmental conditions. Estimated across the whole clade, small but substantial and approximately equal fractions of the euchromatic portion of the genome are inferred to contribute to each of these three sources of adaptive genetic variation. These results indicate that multiple genetic sources can promote rapid diversification and speciation in response to new ecological opportunity, in agreement with our emerging phylogenomic understanding of the complexity of both ancient and recent species radiations.</span>
            </a>
          </div>
          <div class="indented pubcite"><strong>Pease JB</strong>, DC Haak, MW Hahn, &amp; LC Moyle. (In Press). 
          <em>PLoS Biology</em>
          <br />doi: 
          <a href="http://dx.doi.org/10.1371/journal.pbio.1002379"
             class="doilink">10.1371/journal.pbio.1002379</a> &middot; 
         PMID:
         <a class="doilink" href="http://www.ncbi.nlm.nih.gov/pubmed?term=26871574%5Buid%5D">26871574</a>
         &middot; Software: <a href="http://www.github.com/jbpease/mvftools/" target="blank">MVFtools</a></div>
          <div class="pubtitle">
            <a href="http://doi.ieeecomputersociety.org/10.1109/tcbb.2015.2509997">
    Encoding data using biological principles: the Multisample Variant Format for phylogenomics and population genomics
    <span>Rapid progress in the fields of phylogenomics and population genomics has driven increases in both the size of multi-genomic datasets and the number and complexity of genome-wide analyses. We present the Multisample Variant Format, specifically designed to store multiple sequence alignments for phylogenomics and population genomic analysis. The signature feature of MVF is a distinctive encoding of aligned sites with specific biological information content (e.g., invariant, low-coverage). This biological pattern-based encoding allows for rapid filtering and quality control of data and speeds up computation for many analyses. MVF has a simple data structure and flexible header structure to accommodate project metadata, making it an effective data publication and sharing format as well. We also propose variants of the MVF format to accommodate protein and codon alignments, quality scores, and a mix of <em>de novo</em> and reference-aligned data. Using the MVFtools package, MVF files can be converted from other common sequence formats. MVFtools completes tasks ranging from simple transformation and filtering operations to complex genome-wide visualizations in only a few minutes, even on large datasets. In addition to presentation of MVF and MVFtools, we also discuss the application both in MVF and other existing data formats of the broader concept of using biological principles and patterns to inform sequence data encoding.</span></a>
          </div>
          <div class="indented pubcite"><strong>Pease JB</strong> &amp; BK Rosenzweig. (In Press). 
          <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>. 
          <br />doi: 
          <a href="http://dx.doi.org/10.1109/tcbb.2015.2509997"
             class="doilink">10.1109/tcbb.2015.2509997</a> &middot; PMID:
         <a class="doilink" href="http://www.ncbi.nlm.nih.gov/pubmed?term=26701894%5Buid%5D">26701894</a>
          &middot; Software: <a href="http://www.github.com/jbpease/mvftools/" target="blank">MVFtools</a></div>

         <div class="pubtitle">
            <a href="http://sysbio.oxfordjournals.org/content/early/2015/04/17/sysbio.syv023">Detection and polarization of
            introgression in a five-taxon phylogeny. 
            <span>When multiple speciation events occur rapidly in succession, discordant genealogies due to incomplete lineage
            sorting (ILS) can complicate the detection of introgression. A variety of methods, including the 
            <em>D</em>-statistic (a.k.a. the “ABBA-BABA test”), have been proposed to infer introgression in the presence of
            ILS for a four-taxon clade. However, no integrated method exists to detect introgression using allelic patterns for
            more complex phylogenies. Here we explore the issues associated with previous systems of applying 
            <em>D</em>-statistics to a larger tree topology, and propose new 
            <em>D</em> 
            <sub>FOIL</sub> tests as an integrated framework to infer both the taxa involved in and the direction of introgression
            for a symmetric five-taxon phylogeny. Using theory and simulations, we show that the 
            <em>D</em> 
            <sub>FOIL</sub> statistics correctly identify the introgression donor and recipient lineages, even at low rates of
            introgression. 
            <em>D</em> 
            <sub>FOIL</sub> is also shown to have extremely low false positive rates. The 
            <em>D</em> 
            <sub>FOIL</sub> tests are computationally inexpensive to calculate and can easily be applied to phylogenomic datasets,
            both genome-wide and in windows of the genome. In addition, we explore both the principles and problems of
            introgression detection in even more complex phylogenies.</span></a>
          </div>          
           <div class="indented pubcite"><strong>Pease JB</strong> &amp; MW Hahn. 2015. 
                         <em>Systematic Biology</em> 64 (4): 651&ndash;662.
                                   <br />doi: 
                                             <a href="http://dx.doi.org/10.1093/sysbio/syv023"
                                                          class="doilink">10.1093/sysbio/syv023</a> &middot; PMID:
                                                                <a class="doilink" href="http://www.ncbi.nlm.nih.gov/pubmed?term=25888025%5Buid%5D">25888025</a>
                                                                          &middot; Software: <a href="http://www.github.com/jbpease/dfoil/" target="blank">dfoil</a></div>
          <div class="pubtitle">
            <a href="http://dx.doi.org/10.1126/science.1258524">Extensive introgression in a malaria vector species complex
            revealed by phylogenomics. 
            <span>
            <img src="img/science_2015_01_02.gif" height="120" alt="Science Cover" />Introgressive hybridization is now recognized
            as a widespread phenomenon, but its role in evolution remains contested. Here, we use newly available reference genome
            assemblies to investigate phylogenetic relationships and introgression in a medically important group of Afrotropical
            mosquito sibling species. We have identified the correct species branching order to resolve a contentious phylogeny and
            show that lineages leading to the principal vectors of human malaria were among the first to split. Pervasive autosomal
            introgression between these malaria vectors means that only a small fraction of the genome, mainly on the X chromosome,
            has not crossed species boundaries. Our results suggest that traits enhancing vectorial capacity may be gained through
            interspecific gene flow, including between nonsister species.</span></a></div>
          <div class="indented pubcite">Fontaine, MC*, <strong>JB Pease*</strong>, A Steele, RM Waterhouse, DE Neafsey, IV Sharakhov, X Jiang,
                      AB Hall, F Catteruccia, E Kakani, SN Mitchell, Y Wu, HA Smith, RR Love, MK Lawniczak, MA Slotman, SJ Emrich, MW Hahn, NJ Besansky. 2015. 
          <em>Science</em> 347 (6217), 1258524.
          <br />doi: 
          <a href="http://dx.doi.org/10.1126/science.1258524" class="doilink">10.1126/science.1258524</a> &middot; PMID: 
          <a class="doilink" href="http://www.ncbi.nlm.nih.gov/pubmed?term=25431491%5Buid%5D">25431491</a></div>
          <div class="pubtitle">
            <a href="http://dx.doi.org/10.1534/genetics.114.169276">Quantitative genetic analysis indicates natural selection on
            leaf phenotypes across wild tomato species ( 
            <em>Solanum</em> sect. 
            <em>Lycopersicon</em>; Solanaceae) 
            <span>Adaptive evolution requires both raw genetic material and an accessible path of high fitness from one fitness
            peak to another. In this study, we used an introgression line (IL) population to map quantitative trait loci (QTL) for
            leaf traits thought to be associated with adaptation to precipitation in wild tomatoes ( 
            <em>Solanum</em> sect. 
            <em>Lycopersicon</em>; Solanaceae). A QTL sign test showed that several traits likely evolved under directional natural
            selection. Leaf traits correlated across species do not share a common genetic basis, consistent with a scenario in
            which selection maintains trait covariation unconstrained by pleiotropy or linkage disequilibrium. Two large effect QTL
            for stomatal distribution colocalized with key genes in the stomatal development pathway, suggesting promising
            candidates for the molecular bases of adaptation in these species. Furthermore, macroevolutionary transitions between
            vastly different stomatal distributions may not be constrained when such large-effect mutations are available. Finally,
            genetic correlations between stomatal traits measured in this study and data on carbon isotope discrimination from the
            same ILs support a functional hypothesis that the distribution of stomata affects the resistance to CO 
            <sub>2</sub> diffusion inside the leaf, a trait implicated in climatic adaptation in wild tomatoes. Along with evidence
            from previous comparative and experimental studies, this analysis indicates that leaf traits are an important component
            of climatic niche adaptation in wild tomatoes and demonstrates that some trait transitions between species could have
            involved few, large-effect genetic changes, allowing rapid responses to new environmental conditions.</span></a>
          </div>
          <div class="indented pubcite">Muir CD, <strong>JB Pease</strong>, LC Moyle. 2014. 
          <em>Genetics</em> 198:1629&ndash;43.
          <br />doi: 
          <a href="http://dx.doi.org/10.1534/genetics.114.169276" class="doilink">10.1534/genetics.114.169276</a> &middot; PMID: 
          <a class="doilink" href="http://www.ncbi.nlm.nih.gov/pubmed?term=25298519%5Buid%5D">25298519</a></div>
          <div class="pubtitle">
            <a href="http://dx.doi.org/10.1111/evo.12118">More accurate phylogenies inferred from low-recombination regions in the
            presence of incomplete lineage sorting. 
            <span>When speciation events occur in rapid succession, incomplete lineage sorting (ILS) can cause disagreement among
            individual gene trees. The probability that ILS affects a given locus is directly related to its effective population
            size (<em>N<sub>e</sub></em>), which in turn is proportional to the recombination rate if there is strong selection across the
            genome. Based on these expectations, we hypothesized that low-recombination regions of the genome, as well as sex
            chromosomes and nonrecombining chromosomes, should exhibit lower levels of ILS. We tested this hypothesis in
            phylogenomic datasets from primates, the 
            <em>Drosophila melanogaster</em> clade, and the 
            <em>Drosophila simulans</em> clade. In all three cases, regions of the genome with low or no recombination showed
            significantly stronger support for the putative species tree, although results from the X chromosome differed among
            clades. Our results suggest that recurrent selection is acting in these low-recombination regions, such that current
            levels of diversity also reflect past decreases in the effective population size at these same loci. The results also
            demonstrate how considering the genomic context of a gene tree can assist in more accurate determination of the true
            species phylogeny, especially in cases where a whole-genome phylogeny appears to be an unresolvable
            polytomy.</span></a>
          </div>
          <div class="indented pubcite"><strong>Pease JB</strong> &amp; MW Hahn. 2013. 
          <em>Evolution</em> 67:2376&ndash;84.
          <br />doi: 
          <a href="http://dx.doi.org/10.1111/evo.12118" class="doilink">10.1111/evo.12118</a> &middot; PMID: 
          <a class="doilink" href="http://www.ncbi.nlm.nih.gov/pubmed?term=23888858%5Buid%5D">23888858</a></div>
          <div class="pubtitle">
            <a href="http://dx.doi.org/10.1093/molbev/mss010">Sex chromosomes evolved from independent ancestral linkage groups in
            winged insects. 
            <span>The evolution of a pair of chromosomes that differ in appearance between males and females (heteromorphic sex
            chromosomes) has occurred repeatedly across plants and animals. Recent work has shown that the male heterogametic (XY)
            and female heterogametic (ZW) sex chromosomes evolved independently from different pairs of homomorphic autosomes in
            the common ancestor of birds and mammals but also that X and Z chromosomes share many convergent molecular features.
            However, little is known about how often heteromorphic sex chromosomes have either evolved convergently from different
            autosomes or in parallel from the same pair of autosomes and how universal patterns of molecular evolution on sex
            chromosomes really are. Among winged insects with sequenced genomes, there are male heterogametic species in both the
            Diptera (e.g., 
            <em>Drosophila melanogaster</em>) and the Coleoptera ( 
            <em>Tribolium castaneum</em>), female heterogametic species in the Lepidoptera ( 
            <em>Bombyx mori</em>), and haplodiploid species in the Hymenoptera (e.g., 
            <em>Nasonia vitripennis</em>). By determining orthologous relationships among genes on the X and Z chromosomes of
            insects with sequenced genomes, we are able to show that these chromosomes are not homologous to one another but are
            homologous to autosomes in each of the other species. These results strongly imply that heteromorphic sex chromosomes
            have evolved independently from different pairs of ancestral chromosomes in each of the insect orders studied. We also
            find that the convergently evolved X chromosomes of Diptera and Coleoptera share genomic features with each other and
            with vertebrate X chromosomes, including excess gene movement from the X to the autosomes. However, other patterns of
            molecular evolution—such as increased codon bias, decreased gene density, and the paucity of male-biased genes on the
            X—differ among the insect X and Z chromosomes. Our results provide evidence for both differences and nearly universal
            similarities in patterns of evolution among independently derived sex chromosomes.</span></a>
          </div>
          <div class="indented pubcite"><strong>Pease JB</strong> &amp; MW Hahn. 2012. 
          <em>Molecular Biology &amp; Evolution</em> 29: 1645&ndash;53.
          <br />doi: 
          <a href="http://dx.doi.org/10.1093/molbev/mss010" class="doilink">10.1093/molbev/mss010</a> &middot; PMID: 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22319158%5Buid%5D" class="doilink">22319158</a></div>
          <div class="pubtitle">
            <a href="http://dx.doi.org/10.1093/gbe/evr109" class="doilink">No excess gene movement is detected off the Avian or
            Lepidopteran Z chromosome. 
            <span>Most of our knowledge of sex-chromosome evolution comes from male heterogametic (XX/XY) taxa. With the genome
            sequencing of multiple female heterogametic (ZZ/ZW) taxa, we can now ask whether there are patterns of evolution common
            to both sex chromosome systems. In all XX/XY systems examined to date, there is an excess of testis-biased retrogenes
            moving from the X chromosome to the autosomes, which is hypothesized to result from either sexually antagonistic
            selection or escape from meiotic sex chromosome inactivation (MSCI). We examined RNA-mediated (retrotransposed) and
            DNA-mediated gene movement in two independently evolved ZZ/ZW systems, birds (chicken and zebra finch) and
            lepidopterans (silkworm). Even with sexually antagonistic selection likely operating in both taxa and MSCI having been
            identified in the chicken, we find no evidence for an excess of genes moving from the Z chromosome to the autosomes in
            either lineage. We detected no excess for either RNA- or DNA-mediated duplicates, across a range of approaches and
            methods. We offer some potential explanations for this difference between XX/XY and ZZ/ZW sex chromosome systems, but
            further work is needed to distinguish among these hypotheses. Regardless of the root causes, we have identified an
            additional, potentially inherent, difference between XX/XY and ZZ/ZW systems.</span></a>
          </div>
          <div class="indented pubcite">Toups MA, <strong>JB Pease</strong>, MW Hahn. 2011. 
          <em>Genome Biology &amp; Evolution</em> 3:1381&ndash;90.
          <br />doi: 
          <a href="http://dx.doi.org/10.1093/gbe/evr109" class="doilink">10.1093/gbe/evr109</a> &middot; PMID: 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22024813%5Buid%5D" class="doilink">22024813</a></div>
          <div class="pubtitle">
            <a href="http://dx.doi.org/10.1063/1.3455189">On the possible role of robustness in the evolution of infectious
            diseases. 
            <span>Robustness describes the capacity for a biological system to remain canalized despite perturbation. Genetic
            robustness affords maintenance of phenotype despite mutational input, necessarily involving the role of epistasis.
            Environmental robustness is phenotypic constancy in the face of environmental variation, where epistasis may be
            uninvolved. Here we discuss genetic and environmental robustness, from the standpoint of infectious disease evolution,
            and suggest that robustness may be a unifying principle for understanding how different disease agents evolve. We focus
            especially on viruses with RNAgenomes due to their importance in the evolution of emerging diseases and as model
            systems to test robustness theory. We present new data on adaptive constraints for a model RNAvirus challenged to
            evolve in response to UV radiation. We also draw attention to other infectious disease systems where robustness theory
            may prove useful for bridging evolutionary biology and biomedicine, especially the evolution of antibiotic resistance
            in bacteria, immune evasion by influenza, and malaria parasite infections.</span></a>
          </div>
          <div class="indented pubcite">Ogbunugafor CB, <strong>JB Pease</strong>, PE Turner. 2010. 
          <em>Chaos</em> 20(2):026108.
          <br />doi: 
          <a href="http://dx.doi.org/10.1063/1.3455189" class="doilink">10.1063/1.3455189</a> &middot; PMID: 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=20590337%5Buid%5D" class="doilink">20590337</a></div>
          <h3 style="margin-top:20px;">Presentations</h3>
          <ul class="listone">
            <li>Great Lakes Bioinformatics Conference 2015 &middot; Purdue &middot; Best Paper Award</li>
            <li>Midwest Population Genetics 2014 &middot; U. Chicago &middot; Contributed Talk</li>
            <li>Evolution Meeting 2014 &middot; Raleigh &middot; Ernst Mayr Symposium</li>
            <li>Midwest Ecology and Evolution Conference 2014 &middot; Dayton &middot; Contributed Talk</li>
            <li>Evolution Meeting 2013 &middot; Snowbird &middot; Contributed Talk</li>
            <li>Drosophila Research Confeference 2011 &middot; San Diego &middot; Poster</li>
          </ul>
          <h3>Workshops</h3>
          <ul class="listone">
            <li>IU Bioinformatics Clinic &middot; 2014</li>
            <li>Two-day Workshop &middot; Universidad Particular de Loja, Ecuador &middot; 2014</li>
            <li>Seminar on RNA-Seq &middot; Indiana U. &middot; 2014</li>
          </ul>
        </div>
        <div class="three columns">
          <h4 class="nobottom">My Software</h4>
          <div class="softwarelist">
            <ul>
              <li>
                <a href="http://www.github.com/jbpease/dfoil" target="_blank">DFOIL 
                <span>Detection of introgression in five-taxon phylogeny</span></a>
              </li>
                <li>
                <a href="http://www.github.com/jbpease/mixtape" target="_blank">MixTAPE 
                <span>Mix of Tools for Analysis of Phylogenetics and Evolution</span></a>
              </li>
              <li>
                <a href="http://www.github.com/jbpease/mvftools" target="_blank">MVFtools 
                <span>Multisample Variant Format Toolkit</span></a>
              </li>
              <li>
                <a href="http://www.github.com/jbpease/shear" target="_blank">SHEAR 
                <span>Integrated adapter removal, read trimMing, and filtering</span></a>
              </li>
            </ul>
          </div>
        </div>
        <div class="three columns">
          <div class="softwarelist">
            <h4 class="nobottom">Resources</h4>
            <ul>
                            <li>
                <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">BLAST</a>
              </li>
              <li>
                <a href="http://www.flybase.org/" target="_blank">FlyBase</a>
              </li>
              <li>
                <a href="http://genome.jgi.doe.gov/" target="_blank">JGI Genome Portal</a>
              </li>
              <li>
                <a href="https://tree.opentreeoflife.org/" target="_blank">Open Tree of Life</a>
              </li>
              <li>
                <a href="http://www.plantgdb.org/" target="_blank">Plant GDB</a>
              </li>
              <li>
                <a href="http://solgenomics.net/" target="_blank">SolGenomics</a>
              </li>
              <li>
                <a href="http://www.vectorbase.org/" target="_blank">VectorBase</a>
              </li>
            </ul>
          </div>
          <div class="softwarelist">
            <h4 class="nobottom">Other Software</h4>
            <ul>
              <li>
                <a href="https://sites.google.com/site/cmzmasek/home/software/archaeopteryx" target="_blank">Archaeopteryx 
                <span>Versatile phyogeny viewer</span></a>
              </li>
              <li>
                <a href="https://github.com/smirarab/ASTRAL" target="_blank">ASTRAL 
                <span>Coalescent species tree inference.</span></a>
              </li>
              <li>
                <a href="http://www.biopython.org" target="_blank">Biopython 
                <span>Community-driven bioanalytic library for Python.</span></a>
              </li>
              <li>
                <a href="http://www.indiana.edu/~hahnlab/software.html" target="_blank">CAFE 
                <span>Computational analysis of gene family evolution.</span></a>
              </li>
              <li>
                <a href="http://weizhong-lab.ucsd.edu/cd-hit/" target="_blank">CD-HIT 
                <span>Sequence similarity filtering and clustering</span></a>
              </li>
              <li>
                <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC 
                <span>Quality control and diagnostics for short-read sequence</span></a>
              </li>
              <li>
                <a href="http://tree.bio.ed.ac.uk/software/figtree/" target="_blank">FigTree 
                <span>Tree formatting for publication</span></a>
              </li>
              <li>
                <a href="http://main.g2.bx.psu.edu/" target="_blank">Galaxy 
                <span>Web interface for server-based bioinformatic modular workflows</span></a>
              </li>
              <li>
              <a href="http://www.broadinstitute.org/gatk/" target="_blank">GATK 
              <span>Genome Analysis Toolkit</span></a></li> 
              <li> 
              <a href="http://www.micans.org/mcl/" target="_blank">MCL 
              <span>Markov Chain Clustering Program with BLAST integration.</span></a></li>
              <li>
                <a href="http://megasoftware.net/" target="_blank">MEGA 
                <span>Molecular evolution and phylogenetics toolkit</span></a>
              </li>
              <li>
                <a href="http://mesquiteproject.org/mesquite/mesquite.html" target="_blank">Mesquite 
                <span>Suite of modules for phylogenetic analysis.</span></a>
              </li>
              <li>
                <a href="http://home.uchicago.edu/rhudson1/source/mksamples.html" target="_blank">ms 
                <span>Coalescent simulation</span></a>
              </li>
              <li>
                <a href="http://www.drive5.com/muscle/" target="_blank">MUSCLE 
                <span>Multiple sequence alignment</span></a>
              </li>
              <li>
                <a href="http://abacus.gene.ucl.ac.uk/software/paml.html" target="_blank">PAML 
                <span>Phylogenetic Analysis of Maximum Likelihood</span></a>
              </li>
              <li>
                <a href="http://wasabiapp.org/software/prank/" target="_blank">PRANK 
                <span>Multiple sequence alignment</span></a>
              </li>
              <li>
                <a href="http://sco.h-its.org/exelixis/web/software/raxml/index.html" target="_blank">RAxML 
                <span>Phylogenetic tree inference software</span></a>
              </li>
              <li>
                <a href="https://github.com/vsbuffalo/scythe/" target="_blank">Scythe 
                <span>Bayesian 32 adapter trimming</span></a>
              </li>
              <li>
                <a href="http://pbil.univ-lyon1.fr/software/seaview.html" target="_blank">SeaView 
                <span>Sequence Viewer and Editor.</span></a>
              </li>
              <li>
                <a href="https://github.com/spyder-ide/spyder" /target="_blank">Spyder 
                <span>Scientific Python Development Suite</span></a>
              </li>
              <li>
                <a href="https://github.com/alexdobin/STAR/" target="_blank">STAR 
                <span>Spliced RNA-Seq Aligner</span></a>
              </li>
              <li>
                <a href="http://trinityrnaseq.github.io" target="_blank">Trinity 
                <span>
                <em>de novo</em> transcriptome assembly.</span></a>
              </li>
                          </ul>
          </div>
			<div class="softwarelist">
            <h4 class="nobottom">Design</h4>
            <ul>
                <li>
                    <a href="http://www.fontsquirrel.com/" target="_blank">Font Squirrel</a>
                </li>

				<li>
					<a href="https://github.com/hakimel/reveal.js" target="_blank">Reveal.js</a>
				</li>

				<li>
					<a href="https://www.theleagueofmoveabletype.com/" target="_blank">League of Moveable Type</a>
				</li>
			  <li>
                <a href="http://matplotlib.org/" target="_blank">matplotlib 
                <span>Python graphing and visualization library</span></a>
              </li>
              <li>
                <a href="http://phylopic.org/" target="_blank">PhyloPic</a>
              </li>
             <li>
                <a href="http://home.gna.org/veusz/" target="_blank">Veusz 
                <span>Python-based clean figure plotting</span></a>
              </li>
 
			  </div>
        </div>
              
        <div class="three columns">
										<a class="twitter-timeline" href="https://twitter.com/jamesbpease"
					data-widget-id="604522753929003008">Tweets by @jamesbpease</a>

				</div>
      </div>
    </div>
    <!-- container -->
    <!-- End Document
================================================== -->
  </body>
 				<script>
					!function(d, s, id) {
						var js, fjs = d.getElementsByTagName(s)[0], p = /^http:/
								.test(d.location) ? 'http' : 'https';
						if (!d.getElementById(id)) {
							js = d.createElement(s);
							js.id = id;
							js.src = p + "://platform.twitter.com/widgets.js";
							fjs.parentNode.insertBefore(js, fjs);
						}
					}(document, "script", "twitter-wjs");
				</script>
</html>
